packages/shared-models/src/ymodels.ts

```diff
@@ -1031,6 +1050,26 @@ export class YMarkdownCell
       attachments: this.getAttachments()
     };
   }
+
+  /**
+   * Gets whether or not the cell is rendered.
+   *
+   * @returns The cell "rendered" state.
+   */
+  get rendered(): boolean {
```

Comment:
I would change the type to be `boolean` or `null` as only markdown cells have that state.

```suggestion
  get rendered(): boolean | null {
```